components/lock.rst

```diff
@@ -199,6 +199,38 @@ This component also provides two useful methods related to expiring locks:
 ``getRemainingLifetime()`` (which returns ``null`` or a ``float``
 as seconds) and ``isExpired()`` (which returns a boolean).
 
+Automatically Releasing The Lock
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A Lock will be automatically released when the Lock object is destructed. This is
+an implementation detail that will be important when Locks are shared between processes.
+In the example below, the ``pcntl_fork()`` will create two processes and the Lock
+will be released automatically as soon as one process finishes::
+
+    // ...
+    $lock = $factory->createLock('report-generation', 3600);
+    if (!$lock->acquire()) {
+        return;
+    }
+
+    $pid = pcntl_fork();
+    if ($pid == -1) {
+         die('could not fork');
+    } elseif ($pid) {
+         // Parent process
+         sleep(30);
+    } else {
+         // Child process
+         echo 'The lock will be released now.'
+         exit(0);
+    }
+    // ...
+
+
+To disable this behavior, one needs to set the 3 argument to the ``LockFactory::createLock()``
```

Comment:
Type here, i believe it should be `3rd`?

Reply:
Thank you